ansible-ipi-install/roles/installer/defaults/main.yml

```diff
@@ -2,3 +2,46 @@
 # defaults file for installer
 cache_enabled: false
 provision_cache_store: "{{ ansible_env.HOME }}/rhcos_image_cache/"
+
+# Information used to create certificates when creating the registry.
+cert_country: MyCountry
+cert_state: MyState
+cert_locality: MyCity
+cert_organization: MyCompany
+cert_origanizational_unit: MyBusinessUnit
+
+# A file that contained extra auths to be included in the pull secret.
+disconnected_registry_auths_file:
+
+# Name of a file containing the additionalTrustBundle and mirrored
+#   repository settings. If this is set, then a local registry will not be created.
+#   Setting this causes the playbook to use an existing disconnected registry.
+disconnected_registry_mirrors_file:
+
+# The containers registry port
+registry_port_container: 5000
+
+# The port exposed on the server for the registry
+registry_port: "{{ registry_port_container }}"
+
+# The following variables define the registry location
+registry_dir: /opt/registry
+registry_dir_auth: "{{ registry_dir }}/auth"
+registry_dir_cert: "{{ registry_dir }}/certs"
+registry_dir_data: "{{ registry_dir }}/data"
+
+# Name of the pod running as the registry.
+pod_name_registry: ocpdiscon-registry
+
+# The information for the locally created registry
+local_registry: "{{ ansible_fqdn }}:{{ registry_port }}"
+local_repo: ocp4/openshift4
+
+# When a registry is created, then information to connect 
+#   to it is written to the following files.
```

Comment:
remove 2 spaces

Reply:
i do that for readability. but to remain consistent with your comments, I removed the spaces.
pushed